docs/adr/0011-reporting-suspicious-activity.md

```diff
@@ -0,0 +1,98 @@
+# Design for reporting suspicious activity
+
+Establish a pattern for implementing workflows in a resilient and extendable way.  The initial use case will be the report suspicious activity workflow.
+
+## Context
+
+We will allow the user to select one or more records from the activity log to report as suspicious activity.  A suspicious activity report consists of a Zendesk ticket, a TxMA event and an email confirmation.  All of these tasks must finish before a suspicious activity report is considered complete.  Some of the tasks must happen serially and some can be run in parallel.  All of the tasks must be resilient to error. 
+
+The required tasks are:
+
+1. Update the activity log items to show report requested.
+2. Create Zendesk ticket
+3. Update the activity log items with the ticket number
+4. Send an event to TxMA that includes the Zendesk ticket number
+5. Update the activity log items with the TxMA event id
+6. Send an email to Notify.
```

Comment:
As per yesterday's discussion, do we still need the notify steps?

Reply:
No we don't but I think having this documented as part of the process still makes a good case for adopting Step Functions.  And the requirements around the emailing are still changing.

Reply:
Latest update from Ben and TICF is that we're going to switch back to using Notify now, but only send the confirmation email. Let's leave this step in the ADR for now and if we don't end up building it we can consider coming back later and removing it if it's confusing.